Netable/Netable/Netable.swift

```diff
@@ -275,3 +275,5 @@ public actor Netable {
         }
     }
 }
+
+extension AnyPublisher: @unchecked Sendable {}
```

Comment:
I really didn't want to use `@unchecked` here, but if you're implementing it retroactively, you have to. :( I'm not sure if maybe there is a better way to tackle this?